src/sentry/static/sentry/app/components/progressRing.tsx

```diff
@@ -11,6 +11,11 @@ type TextProps = {
   theme: Theme;
 };
 
+export type ProgressColorFunction = {
```

Comment:
Could you call this `ProgressColorFunctionParams` or just make it `ProgressColorFunction = (opts: {
  percent: number;
  theme: Theme;
}) => string;

Reply:
I'd also prefer not to export this if possible, instead your `getCrashFreePercentColor` function can be

```tsx
Type ProgressRingColorFn = React.ComponentProps<ProgressRing>['color'];

export const getCrashFreePercentColor: ProgressRingColorFn = ({
  ...
});
```

Reply:
The reason I say I'd prefer not to export btw is just to keep our API surface small. This is especially helpful with modern autocomplete tooling where having a concise list of imports based on the first little bit of what you've typed is useful, and PrgoressColorFunction is highly specific, and not something we would often import if I started typing `Progress`.

Reply:
Makes sense, thanks. I've updated the PR.